source/admin_checklist.rst

```diff
@@ -33,9 +33,9 @@ Annual Checklist
 Google Hangouts:
 ----------------
 
-1. The OSL has it's own gmail account -- log in through gmail, and make sure that
-on youtube your account is set to the OSL account.  
-2. In google+ go to hangouts (under the little 'home' tab in the upper left) and click 'hangouts on air'
-3. Choose 'Start a hangout on air' and fill in the information
-4. Go to youtube, and in video manager click the 'Live Events' button.  Fill in the same information
-5. When you're ready to go live, Hangouts will open, and you click the "Start hangout" button.
+1. The OSL has it's own official Manatee gmail account for this.  Log in through gmail, and make sure that on youtube your account is set to the OSL manatee gmail account.  
```

Comment:
You should state what the gmail account actually is, i.e `manateeseacow@osuosl.org`.